spec/test_spec.rb

```diff
@@ -1 +1,94 @@
+require 'fileutils'
+require 'pry'
+require 'rake'
 
+RSpec.describe "1" do
+
+  app = Rake.application
+  app.load_rakefile
+
+  before(:all)do
+    app['create'].invoke
+  end
+
+  after(:all)do
+    app['delete'].invoke
+  end
```

Comment:
please re-write it using functions. 
For now we are not going to use Rake tasks